plugin/lspconfig.lua

```diff
@@ -0,0 +1,84 @@
+local version_info = vim.version()
+if version_info.minor < 7 then
```

Comment:
You never know when 1.0 is out 🤡 

```suggestion
if version_info.major < 0 and version_info.minor < 7 then
```

Reply:
`vim.fn.has('nvim-0.8')` is the correct way to check this